pkg/codegen/go/gen.go

```diff
@@ -1575,7 +1575,7 @@ func (pkg *pkgContext) getImports(member interface{}, importsAndAliases map[stri
 		return
 	}
 
-	importsAndAliases["github.com/pulumi/pulumi/sdk/v2/go/pulumi"] = ""
+	importsAndAliases["github.com/pulumi/pulumi/sdk/v3/go/pulumi"] = ""
```

Comment:
I assume these changes to the codegen are intended.

Reply:
yeah this is a update to generate the new code based on the new sdk version